cumulusci/cli/cci.py

```diff
@@ -1066,7 +1071,14 @@ def task_run(config, task_name, org, o, debug, debug_before, debug_after, no_pro
     org, org_config = config.get_org(org, fail_if_missing=False)
     task_config = getattr(config.project_config, "tasks__{}".format(task_name))
     if not task_config:
-        raise TaskNotFoundError("Task not found: {}".format(task_name))
+        raise click.UsageError(
+            "Task not found: {}. Did you mean {}".format(
+                task_name,
+                config.project_config.get_suggested_name(
+                    task_name, config.project_config.tasks
+                ),
+            )
+        )
```

Comment:
You can use project_config.get_task here -- it returns a TaskConfig, so you would just to change the code below to use task_config.config instead of task_config, to keep accessing the raw data.